apps/extension/src/infrastructure/application/providers.tsx

```diff
@@ -1,5 +1,7 @@
 import NiceModal from '@ebay/nice-modal-react';
 import { ReactNode, StrictMode } from 'react';
+import { WithPortal } from '@idriss-xyz/ui/providers/with-portal';
```

Comment:
wdyt about creating index in providers, so it can be imported form `@idriss-xyz/ui/providers`?